packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -50,7 +50,7 @@ - (instancetype)initWithWithFrame:(CGRect)frame
     }];
     NSDictionary<NSString*, id>* settings = args[@"settings"];
     [self applySettings:settings];
-    NSString* initialUrl = args[@"initialUrl"];
+    NSString* initialUrl = [args[@"initialUrl"] isEqual:[NSNull null]] ? nil : args[@"initialUrl"];
```

Comment:
nit: I think the following will be a little more readable:

```objc
NSString* initialUrl = args[@"initialUrl"];
if (initialUrl && initialUrl != [NSNull null]) {
```

Reply:
Fix it 03504e3.